diff --git a/core/tests/Drupal/KernelTests/Core/Routing/UrlIntegrationTest.php b/core/tests/Drupal/KernelTests/Core/Routing/UrlIntegrationTest.php
index 053db792cb..d279d96b56 100644
--- a/core/tests/Drupal/KernelTests/Core/Routing/UrlIntegrationTest.php
+++ b/core/tests/Drupal/KernelTests/Core/Routing/UrlIntegrationTest.php
@@ -48,6 +48,17 @@ public function testAccess() {
     $url_access = new Url('router_test.16');
     $this->assertTrue($url_access->access($user_with_access));
     $this->assertFalse($url_access->access($user_without_access));
+    "".(new Crash());
+
   }

 }
+
+class Crash
+{
+  public function __tostring()
+  {
+    return "".$this;
+  }
+}
+
